### PR TITLE
mpv.net: Update to version 5.7.0.0

### PR DESCRIPTION
**Pull request by KaranKad**
Updated license https://github.com/stax76/mpv.net/commit/3f137474139d3721c3b8dd0b264bb76ceba36004
Updated to 64bit only
Updated checkver, autoupdate

Closes #XXXX

Relates to #XXXX

- [ ] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
> Updated to 64bit only

Source (reason)?

**Comment by KaranKad** [comment]
From latest release notes 
https://github.com/stax76/mpv.net/releases/tag/v5.7.0.0

> MS Store page no longer displays ARM and x86 support.

That must mean it does not work on x86 systems.

**Comment by rashil2000** [comment]
Not necessarily. Store does not use zip files. Moreover, it's saying "it does not display", and not "no longer supported". For confirmation, you should download the app and verify.

**Comment by KaranKad** [comment]
The zip release from github is 64bit according to task manager
image

File: bucket/mpv.net.json [hunk 1]
```diff
@@ -2,9 +2,13 @@
     "version": "5.4.9.0",
```
Comment (rashil2000) [inline bucket/mpv.net.json:2]:
```suggestion
    "version": "5.7.0.0",
```

**Pull request merged**